scripts/updates/utils/termux_pkg_upgrade_version.sh

```diff
@@ -25,6 +25,8 @@ termux_pkg_upgrade_version() {
 	local LATEST_VERSION="$1"
 	local SKIP_VERSION_CHECK="${2:-}"
 	local EPOCH
+	local CURRENT_REF
```

Comment:
Since this is a short assignment it might be nice to inline this into 1 line.
```suggestion
	local CURRENT_REF; CURRENT_REF="$(git rev-parse --abbrev-ref HEAD)"
```
Same goes for `EPOCH`.